tools/topology/sof-icl-rt700.m4

```diff
@@ -6,7 +6,7 @@
 include(`utils.m4')
 include(`dai.m4')
 include(`pipeline.m4')
-include(`alh.m4')
+#include(`alh.m4')
```

Comment:
I forgot to add the file in git... But I can't remember if/why it's needed. I think I did this with @ranj063

````
more platform/common/alh.m4
divert(-1)

dnl ALH related macros

dnl ALH_CONFIG_DATA(type, idx)
define(`ALH_CONFIG_DATA',
`SectionVendorTuples."'N_DAI_CONFIG($1$2)`_tuples" {'
`	tokens "sof_alh_tokens"'
`	tuples."word" {'
`	}'
`	tuples."short" {'
`	}'
`}'
`SectionData."'N_DAI_CONFIG($1$2)`_data" {'
`	tuples "'N_DAI_CONFIG($1$2)`_tuples"'
`}'
)

divert(0)dnl
````

Reply:
According to @bardliao,  alh.m4 is not used.

Reply:
> According to @bardliao, alh.m4 is not used.

Acked. @plbossart IIRC, we did it on Poland. But we didn't use it eventually. We did everything in dai.m4.